Jenkinsfile

```diff
@@ -130,7 +130,10 @@ rocmtest clang_debug: rocmnode('vega') { cmake_build ->
     stage('MLIR Debug') {
         withEnv(['MIGRAPHX_ENABLE_MLIR=1']) {
             def sanitizers = "undefined"
-            def debug_flags = "-g -O2 -fsanitize=${sanitizers} -fno-sanitize-recover=${sanitizers}"
+            // Note: the -fno-sanitize= is copied from upstream LLVM_UBSAN_FLAGS.
+            // This is, as best as I can tell, requirede to build LLVM with
+            // both UBSAN and threads.
+            def debug_flags = "-g -O2 -fsanitize=${sanitizers} -fno-sanitize-recover=${sanitizers} -fno-sanitize=vptr,function"
```

Comment:
Having gone and looked, this could be it, because LLVM explicitly doesn't use C++'s RTTI and instead does its own thing, which could be making UBSAN confused.

Reply:
I dont think there is a function sanitizer. The CI errors with `cc: error: unrecognized argument to ‘-fno-sanitize=’ option: ‘function’`.

Reply:
@pfultz2 What compiler are y'all using for this build? Because upstream LLVM has that exact option